### PR TITLE
add ci build for fedora, centos and debian

### PR DESCRIPTION
**Pull request by pgaubatz**
This is basically the same as https://github.com/mheese/journalbeat/pull/26 but rebased onto master and including a debian (jessie) builder (see https://github.com/mheese/journalbeat/commit/7b60c05bc25dd4e8f950cbe2a4d70b128fd9352d).

### CODE REVIEW

**Comment by langston-barrett** [comment]
@pgaubatz Do you have a link to a log of a working build? Just to make sure :-)

**Comment by pgaubatz** [comment]
Yes, I do. :)
See https://travis-ci.org/pgaubatz/journalbeat/builds/147418583

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - cat ci/${DISTRO}.dockerfile.part ci/common.dockerfile.part > Dockerfile
+  - docker build -t journalbeat-${DISTRO}-packaging .
+  - docker run -v ${PWD}:/go/src/github.com/mheese/journalbeat:rw journalbeat-${DISTRO}-packaging
+  - mv journalbeat journalbeat-${DISTRO}
+
+deploy:
+  skip_cleanup: true
+  provider: releases
+  api_key:
+    secure: cKI6Z927ZVzI3ToFpzoNxwMwynstHdBIRPD+8cMsFozWXqVGZ0fdgVeCbc6S3/iwjXm4SK7fO5qyA98hyTHj3iGtXXrx6maGrFzdZxIqOg3EzZsi7HOpXvMwHb4xBIGX5KerbIiQAozI9UcXnlK6rAbn4ZU5aoC4Abm88pAvN4qLlEdbjA/pAVnrD4jOyGQjv4HaYr/dqsPmzMTT1lY9MWI4FQgAF/jSp3UTq0mFd5zs1AH7yLCEFZqghsWouwbTItMYMcymnn4QsE/fY1rcB7R8nuE2xmPJM5P+oUqLMu/Z7KwEFJVYliv65E7FPwFnZTMVrxlfXupSkPz7RjZAX7kp17h/ctOA5hdyVnYsYyNRPheEN3u8T8zZAnDFG3c5Vp2iKGTpqV5FFCcoDKrxMskFk4F5I1cPVqcMUEBaCJO2TiU5mlvA/39bzOdqjB+dM7RDjJ4+rpsvnci+V7L8/hUMGbrCIcm3lGDnIPsS0Fz3e69Knj8GOeJtZckMfZwCFfz+wza6vOpbjfIKvBGtorEUR19IuJRf2zTDwqsPxmyrucp8j8+UXFkpbt8qgvD+Q3rPdwbP2wEMrAnQRxUnSq2+ccXcmbQ+HAMN6/Xuw+cE+JkCyQ4i/iwzXelZkBUBfvEY15tiM90UUrnTp6EXsswtybdeOXTN5ZjGWg1zVis=
```
Comment (nrvnrvn) [inline .travis.yml:24]: Is it ok and secure to put the key here? @pgaubatz

Comment (langston-barrett) [inline .travis.yml:24]: It's only decryptable in Travis

**Pull request merged**